### PR TITLE
Features/wallet view

### PR DESCRIPTION
**Pull request by lionellbriones**
1. Updated Vuetify to version 2.0.0-beta.5
2. Implemented new Wallet Home design
 - Header
 - Balance Table converted to Cards

### CODE REVIEW

File: app/public/index.html [hunk 1]
```diff
@@ -20,6 +20,8 @@
     <meta name="google-signin-client_id" content="876733105116-i0hj3s53qiio5k95prpfmj0hp0gmgtor.apps.googleusercontent.com" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons" />    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
```
Comment (chaitanyapotti) [inline app/public/index.html]: Duplicate line. pls remove

File: app/src/components/HeaderItem.vue [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
   props: ['icon', 'isSelected', 'to', 'href'],
   computed: {
     computedClass: function() {
-      return !this.isSelected ? 'hidden-sm-and-down' : ''
+      return !this.isSelected ? 'active' : ''
     }
   }
 }
 </script>
+
```
Comment (chaitanyapotti) [inline app/src/components/HeaderItem.vue]: Although, this is a valid style, our style guide uses template, script, style in that order
For more style guides, refer to https://vuejs.org/v2/style-guide/

File: app/package.json [hunk 3]
```diff
@@ -118,6 +118,7 @@
     "nyc": "^14.1.1",
     "prerender-spa-plugin": "^3.4.0",
     "puppeteer": "^1.18.1",
+    "sass": "^1.17.4",
```
Comment (chaitanyapotti) [inline app/package.json]: Why is this necessary? Was it mentioned in the vuetify docs.?
If so, pls update it to the latest version

File: app/package.json [hunk 4]
```diff
@@ -69,7 +69,7 @@
     "vue": "^2.6.10",
     "vue-knob-control": "^1.6.0",
     "vue-router": "^3.0.7",
-    "vuetify": "^1.5.16",
+    "vuetify": "^2.0.0-beta.5",
```
Comment (chaitanyapotti) [inline app/package.json]: Still more errors in console for vuetify. pls resolve them

Comment (lionellbriones) [reply app/package.json]: > Still more errors in console for vuetify. pls resolve them
Done.

File: app/src/components/HeaderItem.vue (hunk 2 above)
Comment (lionellbriones) [reply app/src/components/HeaderItem.vue]: > Although, this is a valid style, our style guide uses template, script, style in that order
> For more style guides, refer to https://vuejs.org/v2/style-guide/

Done.

File: app/public/index.html (hunk 1 above)
Comment (lionellbriones) [reply app/public/index.html]: > Duplicate line. pls remove

Done.

File: app/src/components/HeaderItem.vue (hunk 2 above)
Comment (lionellbriones) [reply app/src/components/HeaderItem.vue]: > Although, this is a valid style, our style guide uses template, script, style in that order
> For more style guides, refer to https://vuejs.org/v2/style-guide/

Done.

File: app/package.json (hunk 3 above)
Comment (lionellbriones) [reply app/package.json]: > Why is this necessary? Was it mentioned in the vuetify docs.?
> If so, pls update it to the latest version

Yes this is mentioned on the docs. I updated the version to 1.22.3

File: app/package.json (hunk 4 above)
Comment (lionellbriones) [reply app/package.json]: > Still more errors in console for vuetify. pls resolve them

Done

File: app/public/index.html [hunk 5]
```diff
@@ -19,7 +19,8 @@
     <script src="<%= BASE_URL %>js/gapi.js" crossorigin="anonymous" integrity="sha384-m0UrUl+xP4P/jooB7I71mIMQv8pSXopfvry4S8m1KJZTKCLv5wVy1h9JIADN2v8T"></script>
     <meta name="google-signin-client_id" content="876733105116-i0hj3s53qiio5k95prpfmj0hp0gmgtor.apps.googleusercontent.com" />
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900" />
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons" />    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css" />
```
Comment (chaitanyapotti) [inline app/public/index.html]: please download the icons used and put them in the public folder

File: app/src/containers/WalletHome.vue [hunk 6]
```diff
@@ -76,15 +116,25 @@ export default {
...
     finalBalancesArray() {
-      return this.$store.getters.tokenBalances.finalBalancesArray || []
+      let balances = this.$store.getters.tokenBalances.finalBalancesArray
+
+      return balances || []
+    },
+    filteredBalancesArray() {
+      const search = this.search || ''
+      return this.finalBalancesArray.filter(balance => {
+        return balance.name.toLowerCase().indexOf(search.toLowerCase()) >= 0
```
Comment (chaitanyapotti) [inline app/src/containers/WalletHome.vue]: use regex

File: app/src/containers/WalletHome.vue [hunk 7]
```diff
@@ -113,11 +166,17 @@ export default {
...
-      this.$router.push({ path: '/wallet/transfer', query: { address: this.selected[0].tokenAddress.toLowerCase() } })
+      // this.$router.push({ path: '/wallet/transfer', query: { address: this.selected[0].tokenAddress.toLowerCase() } })
+      this.$router.push({ path: '/wallet/transfer' })
     },
     topup() {
       this.$router.push({ path: '/wallet/topup' })
     }
+  },
+  watch: {
+    finalBalancesArray(value) {
```
Comment (chaitanyapotti) [inline app/src/containers/WalletHome.vue]: can use this as a computed property instead. ideally use watchers only for async calls

File: app/src/main.js [hunk 8]
```diff
@@ -6,6 +6,7 @@ import router from './router'
 import store from './store'
 import log from 'loglevel'
 import './registerServiceWorker'
+import vuetify from './plugins/vuetify'
```
Comment (chaitanyapotti) [inline app/src/main.js]: Use the already imported plugins to bring in the vuetify object. don't do it twice.

File: app/public/index.html (hunk 5 above)
Comment (lionellbriones) [reply app/public/index.html]: Done

File: app/src/containers/WalletHome.vue (hunk 6 above)
Comment (lionellbriones) [reply app/src/containers/WalletHome.vue]: Done

File: app/src/containers/WalletHome.vue (hunk 7 above)
Comment (lionellbriones) [reply app/src/containers/WalletHome.vue]: Done.. moved to computed property

File: app/src/main.js (hunk 8 above)
Comment (lionellbriones) [reply app/src/main.js]: Done. I also exported vuetify in plugin/index.js as it needs to be bootstrapped on main Vue instance

File: app/public/index.html (hunk 5 above)
Comment (chaitanyapotti) [reply app/public/index.html]: Also, delete any unused icons in public/images folder (before merging.. not right now).
Don't delete any icons present in logos folder

File: app/src/views/Confirm.vue [hunk 9]
```diff
@@ -329,6 +416,25 @@ export default {
...
+      const currencyMultiplier = this.getCurrencyMultiplier
+      const ethFee = this.gasEstimate * fastGasPrice * 10 ** -9
+      const currencyFee = ethFee * currencyMultiplier
+      return `${significantDigits(currencyFee)} ${this.$store.state.selectedCurrency}`
+    },
+    getSelectedSpeed() {
+      let selectedType = ''
+      let nearest = 1000
+      const speedOptions = [...this.speedOptions]
+      speedOptions.reverse().forEach(speed => {
```
Comment (chaitanyapotti) [inline app/src/views/Confirm.vue]: why reverse?

File: app/src/views/Confirm.vue [hunk 10]
```diff
@@ -329,6 +416,25 @@ export default {
...
+    getGasDisplayString(speed, fastGasPrice) {
+      const currencyMultiplier = this.getCurrencyMultiplier
+      const ethFee = this.gasEstimate * fastGasPrice * 10 ** -9
+      const currencyFee = ethFee * currencyMultiplier
+      return `${significantDigits(currencyFee)} ${this.$store.state.selectedCurrency}`
+    },
+    getSelectedSpeed() {
+      let selectedType = ''
+      let nearest = 1000
+      const speedOptions = [...this.speedOptions]
```
Comment (chaitanyapotti) [inline app/src/views/Confirm.vue]: not necessary. .reverse() returns a new array and doesn't mutate original

Comment (lionellbriones) [reply app/src/views/Confirm.vue]: In case there are options with same gasPrice then I'll get the one with faster speed. I didn't intend to mutate the original so the layout would stay the same..
